packages/cosmwasm-schema-cli/package.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "@satlayer/cosmwasm-schema-cli",
+  "version": "0.0.0",
+  "private": true,
```

```suggestion
  "private": false,
```

- `version` should be removed, since it's generated by the release.yml
- Should be set to private: false